setup.py

```diff
@@ -15,6 +15,24 @@
 import versioneer
 
 
+def install_custom_ops() -> bool:
+    """Determine if we should install the custom ops.
+
+    If it is not a linux machine and doesn't have at least nvcc we skip it. Can
+    still use the reference platform if needed in such cases.
+    """
+    if os.environ.get("SKIP_CUSTOM_OPS"):
+        return False
+    if "linux" not in sys.platform:
+        return False
+    try:
+        subprocess.check_call(["nvcc"])
```

Comment:
This fails for me locally because `nvcc` returns a failure status when passed no files. `nvcc --version` seems to work for me, though.